remote_attestation/rust/src/handshaker.rs

```diff
@@ -537,6 +584,20 @@ pub struct AttestationBehavior {
     signer: Option<Signer>,
 }
 
+impl std::fmt::Debug for AttestationBehavior {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        match (
+            self.contains_peer_attestation(),
+            self.contains_self_attestation(),
+        ) {
+            (true, false) => write!(f, "PeerAttestation"),
+            (false, true) => write!(f, "SelfAttestation"),
+            (true, true) => write!(f, "BidirectionalAttestation"),
+            _ => write!(f, "InvalidAttestation"),
```

Comment:
nit: Perhaps `(false, false)` is clearer.

Reply:
Done